data_steward/retraction/retract_deactivated_pids.py

```diff
@@ -0,0 +1,441 @@
+# Python imports
+import re
+import argparse
+import logging
+
+# Third party imports
+import pandas as pd
+
+# Project imports
+from utils import bq
+import bq_utils
+from retraction.retract_utils import DEID_REGEX
+from cdr_cleaner.cleaning_rules import sandbox_and_remove_pids
+from constants.cdr_cleaner import clean_cdr as clean_consts
+from sandbox import get_sandbox_dataset_id, create_sandbox_dataset
+from constants import bq_utils as bq_consts
+
+DEACTIVATED_PIDS_QUERY = """
```

Comment:
I was recently introduce to the jinja templates.  I use one here:  https://github.com/all-of-us/curation/blob/develop/tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py#L44

It allows us to format the sql nicely in code without sending a lot of useless blank characters through to the API.